cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,44 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
+describe('Article flow', () => {
+  let user;
+  let article;
 
-  });
+  beforeEach(() => {
+    cy.visit('/');
 
-  it('', () => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
```

Comment:
```
    cy.task('generateUser').then((generateUser) => {
      user = generateUser;
```
in your tests you use "generateUser (assigned value "user") - but this is not defined anywhere. Pay attention, how do you made it with generateArticle()

Reply:
Here I have defined  "generateUser". It also was written before me, in the homework that I forked.
  on('task', {
        generateUser() {
          const email = faker.internet.email();
          const randomNumber = Math.floor(Math.random(1000) * 1000);
          return {
            username: faker.name.firstName() + randomNumber,
            username: faker.person.firstName() + randomNumber,
            email: email.toLowerCase(),
            password: '12345Qwert!'
          };
        }
      });